### PR TITLE
[docs/developers] prepare for submission

### PR DESCRIPTION
**Pull request by pdiercks**
Hi @dglaeser, Hi @joergfunger ,
I came across and added another [list of workflow tools](https://github.com/pditommaso/awesome-pipeline) to the docs. I opened this PR to discuss how we can finalise/limit the selection of tools for implementation of the test-use-case. One idea could be to go by popularity if some form of measure is available. For example, for projects hosted on GitHub one could compare number of watchers, stars and forks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +43,17 @@ reusable solutions and identify the capabilities that are missing to reach our g
...
 
 [Go to the documentation](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/).
+
+## How to contribute
+We greatly appreciate and encourage contributions by any users or developers of 
+workflow tools. 
+A list of preselected tools can be found in the [documentation](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/).
+If you would like to add an implementation of the [simple use case](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/docs/simpleusecase.html) for a tool
+(not limited to the given list), or if you suggest changes to one of the
+existing implementations, please raise an issue and submit a pull request.
```
Comment (dglaeser) [inline README.md:53]:
```suggestion
existing implementations, please raise an issue and/or submit a pull request.
```

File: docs/simpleusecase.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+on a square domain :math:`\varOmega={(0, s)}^2, s\in\mathbb{R}^+`.
+
+.. math::
+    -\Delta u &= f\,,\quad&&\mathrm{in}\;\varOmega\,,\\
+   u &= u_{\mathrm{D}}\,,\quad&&\mathrm{on}\;\partial\varOmega\,.
+
+The domain size :math:`s` is defined as an input to the workflow, such that the workflow can
+be run for different computational domains.
+The final output of the workflow is a PDF document showing the solution :math:`u` over the
+line :math:`L=\{(x, y)\in\varOmega\,\vert\,x=y\}` and the number of degrees of freedom in the finite element model.
```
Comment (dglaeser) [inline docs/simpleusecase.rst:17]:
```suggestion
line :math:`L=\{(x, y)\in\varOmega\,\vert\,x=y\}` and the number of degrees of freedom used in the finite element model.
```

File: docs/simpleusecase.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    :alt: simple_use_case_dag
+
+1. Partition of the computational domain using `Gmsh <http://gmsh.info/>`_, |DAG|
+
+.. code-block:: console
+
+   gmsh -2 -setnumber domain_size 1.0 source/square.geo -o ./square.msh
+
+
+2. Conversion of the file format (into one readable by `FEniCS`) using `meshio <https://github.com/nschloe/meshio>`_,
```
Comment (dglaeser) [inline docs/simpleusecase.rst:32]:
```suggestion
2. Conversion of the file format (into one readable by `FEniCS <https://fenicsproject.org/>`) using `meshio <https://github.com/nschloe/meshio>`_,
```

File: simple_use_case/README.md [hunk 4]
```diff
@@ -1,25 +1,11 @@
...
-the file `conda_env.yml` containing all packages and their versions.
-```
-conda env create --file conda_env.yml --name <env> --prefix <your-prefix>
-```
-The file `conda_info.txt` provides additinoal information about the `conda` version used.
+1. Partition of the computational domain using [Gmsh](http://gmsh.info/),
+2. Conversion of the file format (into one readable by `FEniCS`) using [meshio](https://github.com/nschloe/meshio),
+3. Solution of the poisson equation using [FEniCS](https://fenicsproject.org/),
+4. Postprocessing using [ParaView](https://www.paraview.org/),
+5. Generation of a PDF using [LaTeX](https://www.latex-project.org/).
```
Comment (dglaeser) [inline simple_use_case/README.md:10]: should we link to the more extensive documentation in the .rst file or the generated page?

**Comment by dglaeser** [comment]
Thanks @pdiercks, reads very nice!

**Comment by pdiercks** [comment]
Hi @joergfunger ,
maybe this slipped past you because I did not request review from you directly.
I think we could merge this. The (mostly minor) changes are described above.

File: README.md [hunk 5]
```diff
@@ -43,3 +43,17 @@ reusable solutions and identify the capabilities that are missing to reach our g
...
+A list of preselected tools can be found in the [documentation](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/).
+If you would like to add an implementation of the [simple use case](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/docs/simpleusecase.html) for a tool
+(not limited to the given list), or if you suggest changes to one of the
+existing implementations, please raise an issue and/or submit a pull request.
+We also encourage to start a new [discussion](https://github.com/BAMresearch/NFDI4IngScientificWorkflowRequirements/discussions)
+for general comments, questions and ideas you would like to share.
+
+## Acknowledgements
+We would like to thank the following people for their contribution to this repository:
+* TBA
```
Comment (joergfunger) [inline README.md:59]: either remove this or add our names (but I guess this is already documented within github)

File: docs/simpleusecase.rst [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    cp source/paper.tex ./paper.tex
+    tectonic ./paper.tex 
+
+It is important to note that the `source files <https://github.com/BAMresearch/NFDI4IngScientificWorkflowRequirements/blob/main/simple_use_case/source>`_ are
+regarded as static inputs, i.e. changing them may break the workflow and thus
+only the variable domain size should be exposed as an actual input parameter.
+The number of degrees of freedom is written to stdout
+by the FEniCS script `source/poisson.py`
+and used here to showcase how one might deal with output of integer type (i.e. non-file output).
+If the tool does not support this the number of degrees of freedom is written to file.
```
Comment (joergfunger) [inline docs/simpleusecase.rst:75]: to a file

File: docs/tools.rst [hunk 7]
```diff
@@ -1,12 +1,13 @@
 .. _developers:
 
-List of developers
-==================
-List of developers of relevant workflow tools and how we may get in contact with them.
+List of tools
+=============
+This is a list of preselected tools based on possible requirements at the start of
+the project. Examples of these requirements are, the seamless integration of
+HPC systems or the execution of containerized applications.
```
Comment (joergfunger) [inline docs/tools.rst:7]: Not sure if I would add the HPC integration here as an example, in particular with respect to the last discussions.

**Review by joergfunger — APPROVED** [review]
nice work

File: README.md (hunk 5 above)
Comment (pdiercks) [reply README.md:59]: I added this section to list people that might contribute by reviewing our implementations (i.e. the developers of the tools)
and for the (probably more rare case) that someone adds a implementation of the simple use case for some tool in the future

File: docs/tools.rst (hunk 7 above)
Comment (pdiercks) [reply docs/tools.rst:7]: I removed the sentence.

File: docs/simpleusecase.rst (hunk 6 above)
Comment (pdiercks) [reply docs/simpleusecase.rst:75]: Thanks. Changed accordingly.

File: README.md (hunk 5 above)
Comment (pdiercks) [reply README.md:59]: @joergfunger I would keep this section and add later on the people contributing other implementations of the simple use case (kadi4mat / kadi studio, gnu guix).

Comment (joergfunger) [reply README.md:59]: but isn't that that also seen in github, if they make a pull request? This way, we mention all others, but our contribution is not mentioned at all.

**Review by joergfunger — COMMENTED** [review]
For me it's ready to merge (apart for the discussion related to the acknowledgements)

File: README.md (hunk 5 above)
Comment (pdiercks) [reply README.md]: okay, I removed the section. I agree that every contribution is already documented within github. So no need to list manually.

**Pull request merged**